aiidalab_widgets_base/computational_resources.py

```diff
@@ -229,15 +232,19 @@ def _setup_new_code(self, _=None):
                     "width": "500px",
                     "border": "1px solid gray",
                 }
-                display(
-                    ipw.HTML(
-                        """Please select the computer/code from a database to pre-fill the fields below."""
-                    ),
-                    self.comp_resources_database,
-                    self.setup_message,
-                    self.ssh_computer_setup.password_box,
-                    self.output_tab,
+                children = (
+                    [
+                        ipw.HTML(
+                            """Please select the computer/code from a database to pre-fill the fields below."""
+                        ),
+                        self.comp_resources_database,
+                        self.ssh_computer_setup.password_box,
+                    ]
+                    if self.comp_resources_database.database
+                    else []
                 )
+                children += [self.setup_message, self.output_tab]
```

Comment:
Not sure about this message and output also for the 'detailed setup', if that is the case, I think we need to decouple it from here. But not necessary in this PR.

Reply:
@unkcpz not sure what you mean, but `self.setup_message` is used to display status messages and errors from Detailed Setup as well.

Reply:
Nevermind my comment, I misunderstood about this code, I thought it is for quicksetup only.